src/python/WMCore/REST/Server.py

```diff
@@ -320,7 +320,7 @@ def static(self, *args, **kwargs):
 
         Serves assets as documented in :meth:`_serve`."""
         if len(args) > 1 or (args and args[0] != "yui"):
```

Comment:
I am not quite under stand this condition (although this is not what you changed). are we just supporting multiple file loading only when "yui" keyword is provided in the arguments?